### DISCUSSION TITLE
Reaction.sent_by_bot

### DISCUSSION THREAD

**Discussion by BIRD-UP73** [Rapptz/discord.py#6838] [Ideas]
It would be nice to have a function/property called `Reaction.sent_by_bot` so you do not have to do this everytime:

```py
async def on_reaction_add(reaction: Reaction, user: Union[Member, User]):
    if user == self.bot.user:
        return
```

Don't know if this has been suggested before or I am doing something wrong.

**Comment by Rapptz** [comment]
This is provided by the API under a different name, [`Reaction.me`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Reaction.me).

**Reply by BIRD-UP73** [reply]
I asked about this in the Discord, but `Reaction.me` always seems to be set to true once the bot itself has reacted regardless of who reacts

small code sample:
```py
async def on_reaction_add(self, reaction: Reaction, user: Union[Member, User]):
    logging.warning(f'user: {user}, me {reaction.me}')
```

![loggingbot](https://user-images.githubusercontent.com/57305556/116860143-671f9580-ac01-11eb-9fd9-b07da1cfae5c.png)

**Reply by Rapptz** [reply]
Yeah. it's `True` if you reacted to the message at all. 

I have no plans to add the original feature request. It's unpythonic.

**Reply by BIRD-UP73** [reply]
In that case, would it be possible to add that to the current docstring for [`Reaction.me`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Reaction.me) ?
Because I wasn't sure what 
> If the user sent this reaction.

means.
